source/views/guides/getting-started.html.slim

```diff
@@ -250,11 +250,20 @@ main
                   }
 
         markdown:
-          We'll also want to create a `Post` struct into which we can read our
-          data, and have diesel generate the names we'll use to reference tables
-          and columns in our queries.
+          We'll also want to create two things:
+ 
+          1. `Post` struct for a query, into which we can read our data.
```

Comment:
I think we should clarify here which of those two types refer to the `table!` macro and which to the `Queryable` struct. Maybe by using `posts` instead of `Post` for the table type?

Reply:
I agree it would be good to clarify. Not sure what you mean with `posts` vs. `Post`, but I trust you... go for it! :)

Reply:
Just to clarify that: With `posts` I mean `crate::schema::posts` and with `Post` I mean the struct deriving `Queryable`.